app/assets/javascripts/desktop-protocol-handler.js

```diff
@@ -0,0 +1,44 @@
+(function() {
+
+  var timeIndex
+
+  window.addEventListener("blur", function(e)
+  {
+    clearTimeout(timeIndex);
```

Comment:
If we get the `blur` event on the window, we assume that the browser successfully opened the app and clear the timeout for redirecting to the desktop landing page. This is similar to the approach that Slack uses and I learned about it from here: https://github.com/electron/fiddle/issues/43#issuecomment-409032237